app/backend/src/couchers/models.py

```diff
@@ -1448,6 +1450,10 @@ class HostRequest(Base):
             "ix_host_requests_surfer_didnt_meetup",
             surfer_reason_didnt_meetup != None,
         ),
+        Index(
+            "ix_host_pending_requests",
+            status == HostRequestStatus.pending,
```

Comment:
You could also add `host_sent_request_reminders` and `start_time` here if you wish; basically all the column you're using in the query.